composer.json

```
@@ -29,6 +29,11 @@
 			"settings/"
 		]
 	},
+	"autoload-dev": {
+		"classmap": [
+			"tests/phpunit/includes/"
```

```suggestion
			"tests/phpunit/"
```
Let's autoload all test classes to be able to extend one from another in different folders, what do you think?

---

OK. Not useful for now but may be in the future.
Done in [616794b](https://github.com/polylang/polylang/pull/1325/commits/616794b2f9d955ac2938e4eedfb29c8bff31d6d0).